scripts/deploy-kubevirt-and-cdi.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -e
+
+KUBEVIRT_NIGHTLY=$(curl -L https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/latest)
+KUBEVIRT_NAMESPACE=${1:-kubevirt}
+
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${KUBEVIRT_NAMESPACE}
+EOF
+
+# Deploying kubevirt
+kubectl apply -n ${KUBEVIRT_NAMESPACE} -f "https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${KUBEVIRT_NIGHTLY}/kubevirt-operator.yaml"
+
+kubectl apply -n ${KUBEVIRT_NAMESPACE} -f "https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${KUBEVIRT_NIGHTLY}/kubevirt-cr.yaml"
+
+# Patch to enable needed functionality
+kubectl patch -n ${KUBEVIRT_NAMESPACE} kubevirt kubevirt --type='json' -p '[{
+  "op": "add",
+  "path": "/spec/configuration/developerConfiguration/featureGates/-",
+  "value": "DataVolumes",
+},{
+  "op": "add",
+  "path": "/spec/configuration/developerConfiguration/featureGates/-",
+  "value": "CPUManager",
+},{
+  "op": "add",
+  "path": "/spec/configuration/developerConfiguration/featureGates/-",
+  "value": "KubevirtSeccompProfile",
+},{
+  "op": "add",
+  "path": "/spec/configuration/developerConfiguration/featureGates/-",
+  "value": "NUMA",
+},{
+  "op": "replace",
+  "path": "/spec/configuration/seccompConfiguration",
+  "value": {
+    "virtualMachineInstanceProfile": {
+      "customProfile": {
+        "localhostProfile": "kubevirt/kubevirt.json"
+      }
+    }
+  },
+}]'
```

Comment:
None of them should be needed anymore.

Reply:
To ensure that the function tests run successfully, both NUMA and GPU are required. However, I have removed the others.